<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScheduleTabWidget</class>
 <widget class="QWidget" name="ScheduleTabWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>254</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>7</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QStackedWidget" name="scheduleStack">
     <widget class="QWidget" name="page">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="PreviousWeekButton">
             <property name="maximumSize">
              <size>
               <width>40</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>&lt;&lt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="weekLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>250</width>
               <height>40</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="nextWeekButton">
             <property name="maximumSize">
              <size>
               <width>40</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>&gt;&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="printButton">
             <property name="text">
              <string>Print</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableWidget" name="scheduleTable">
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <row>
            <property name="text">
             <string>9:30-11:00</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string/>
            </property>
           </row>
           <row>
            <property name="text">
             <string>11:00-12:30</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string/>
            </property>
           </row>
           <row>
            <property name="text">
             <string>12:30-14:00</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string/>
            </property>
           </row>
           <row>
            <property name="text">
             <string>14:00-15:30</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string/>
            </property>
           </row>
           <row>
            <property name="text">
             <string>15:30-17:00</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string/>
            </property>
           </row>
           <column>
            <property name="text">
             <string>Monday</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Tuesday</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Wednesday</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Thursday</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Friday</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
